openedx/features/partners/tests/test_helpers.py

```diff
@@ -32,22 +32,6 @@ def setUp(self):
         self.partner_user = PartnerUserFactory(user=self.user, partner=self.partner)
         self.course = CourseFactory.create()
 
-    def test_import_form_using_slug_with_valid_slug(self):
-        """
-        Test if form is available for a valid slug
-        :return : partner view
-        """
-        form = helpers.import_form_using_slug(self.PARTNER_SLUG)
-        self.assertIsNotNone(form)
-
-    def test_import_form_using_slug_with_invalid_slug(self):
-        """
-        Test None is returned for an invalid partner slug
-        :return : None
-        """
-        form = helpers.import_form_using_slug('invalid')
-        self.assertIsNone(form)
```

Comment:
Why did we remove the `invalid_slug` method? Even after removing the `valid_slug` one because we can't use it at this point, this one was running perfectly and acted as a safety net for that helper, that if anything breaks or that helper throws an error, this test fails.

Reply:
Removed it because it doing the same importing the module which structure in we didn't using now and yes you're right about this test result, i again added it 👍